docs/UserGuide.adoc

```diff
@@ -190,15 +147,21 @@ Format: `list c\[classmate/teammate]`
 
 Example:
 
-* `list c\classmate` +
-Lists all your contacts in the `classmate` category.
+* Suppose you want to view all the classmate contacts that you have added into NOVA,
++
+`list c\classmate`
++
+NOVA will lists all your contacts in the `classmate` category as seen below.
 
-* `list c\teammate` +
-Lists all your contacts in the `teammate` category.
+* Suppose you want to view all the teammate contacts that you have added into NOVA,
++
+`list c\teammate`
++
+NOVA will lists all your contacts in the `teammate` category as seen below.
 
 ==== Find Contact: `find`
 You can find a contact added to the address book easily with a name.
-You can find contact by full name, or by first name or last name only.
+You are able to find contact by full name, or by first name or last name only.
```

Comment:
Maybe can connect both sentences into "You can find a contact added to the address book easily, either with the full name or just with the person's first or the last name."